README.md

```diff
@@ -28,7 +28,7 @@ for moving] an existing package to this repository.
 
 - Stable, or backward compatible, API, with complete godocs.
 
-- Go tools compliant (`go get`, `go test`, etc.).
+- Go tools compliant (`go install`, `go test`, etc.).
```

Comment:
Let's keep `go get` in this list along with `go install`. Folks might want to use it to resolve dependencies.

Reply:
@nitishfy this is not resolved. The list should have all 3: `go get`, `go install` and `go test`.